web/js/layers/layers.add.js

```diff
@@ -1,13 +1,10 @@
-/**
- * @module wv.layers
- */
-var wv = wv || {};
-wv.layers = wv.layers || {};
-
-/**
- * @class wv.layers.add
- */
-wv.layers.add = wv.layers.add || function (models, ui, config) {
+import $ from 'jquery';
+import each from 'lodash/each';
+import find from 'lodash/find';
+import throttle from 'lodash/throttle';
+import util from '../util/util';
+
+export function layersAdd(models, ui, config) {
```

Comment:
What do you think about keeping the name `add`, importing this into the `layers` parent, and attaching it to the object that exports?

Reply:
Cool. I can do that, I like that structure.